docs/dev/table/sql.md

```diff
@@ -340,18 +567,14 @@ val result4 = tableEnv.sql(
 
 {% top %}
 
-### Limitations
-
-**TODO: Integrate this with the examples**
-
 #### Batch
```

Comment:
I would remove the `Batch` and `Streaming` limitations sections. 
Instead, I would integrate the relevant information with the `Operations` sections, e.g., add a comments to UNNEST that WITH ORDINALITY is not supported, etc.